ia-configs/IA.amphibian-reptile.json

```diff
@@ -1,50 +1,50 @@
-    "_default": "@Salamandra.salamandra",
-    "Salamandra": {
-      "salamandra": {
-        "_default":{
-          "_id_conf": "@Salamandra.salamandra.fire_sal._id_conf"
-        },
-        "_detect_conf": [
+  "Salamandra": {
+    "salamandra": {
+      "_default": {
+        "_id_conf": "@Salamandra.salamandra.fire_sal._id_conf"
+      },
+      "_detect_conf": [
+        {
+          "labeler_algo": "densenet",
+          "labeler_model_tag": "salanader_fire_v0",
+          "model_tag": "salanader_fire_v0",
+          "nms_thresh": 0.4,
+          "sensitivity": 0.5
+        }
+      ],
+      "fire_sal": {
+        "_id_conf": [
           {
-            "labeler_algo": "densenet",
-            "model_tag": "salanader_fire_v0",
-            "labeler_model_tag": "salanader_fire_v0",
-            "nms_thresh": 0.40,
-            "sensitivity": 0.50
-          }
-        ],
-        "fire_sal":{
-          "_id_conf": [
-            {
-              "query_config_dict": {
-                "sv_on":true
-              },
-              "description": "HotSpotter pattern-matcher"
+            "description": "HotSpotter pattern-matcher",
+            "query_config_dict": {
+              "sv_on": true
             }
-          ]
-        },
-        "salanader_fire": {
-          "_id_conf": [
-            {
-              "query_config_dict": {
-                "sv_on": true
-              },
-              "description": "HotSpotter pattern-matcher"
+          }
+        ]
+      },
+      "salanader_fire": {
+        "_id_conf": [
+          {
+            "description": "HotSpotter pattern-matcher",
+            "query_config_dict": {
+              "sv_on": true
             }
-          ]
-        },
-        "salanader_fire_adult": {
-          "_id_conf": [
-            {
-              "query_config_dict": {
-                "sv_on": true
-              },
-              "description": "HotSpotter pattern-matcher"
+          }
+        ]
+      },
+      "salanader_fire_adult": {
+        "_id_conf": [
+          {
+            "description": "HotSpotter pattern-matcher",
+            "query_config_dict": {
+              "sv_on": true
             }
-          ]
-        }
+          }
+        ]
       }
     }
-  }
+  },
+  "_default": "@Salamandra.salamandra",
+  "_note": "The hierarchy of this file is genus->species->ia_class->_id_conf (id config). detect_config is under Genus->species. Keys with a leading underscore and their children are non-biological categories of config, like _id_conf and _detection_conf. Keys without leading underscores are semantic lookups that are derived from Wildbook data, e.g. Megaptera.novaeangliae.whale_fluke."
+}
```

Comment:
Since these notes are meant to be read by a programmer opening the config file I prefer them at the top. Not a big deal.

Reply:
The pre-commit options for [`pretty-format-json`](https://gdevops.gitlab.io/tuto_git/tools/pre-commit/repos_hooks/repo_pre_commit_hooks/pretty-format-json/pretty-format-json.html) has an argument for `--top-keys`.   This argument allows me to sort the list of keys but specify a list of keys to keep near the top.  I've added `_README` and `_note`.